pkg/util/util.go

```diff
@@ -365,3 +366,12 @@ func AddVMIObjectGraph(spec v1.VirtualMachineInstanceSpec, namespace string, ext
 	return extra
 
 }
+
+func IsMacAdressClearedByAnnotation(vm *v1.VirtualMachine) (bool) {
+	annotations := vm.GetAnnotations()
```

Comment:
It may make more sense to have the annotation in the restore object so the decision about clearing mac address out can be made retroactively.